temporalio/client.py

```diff
@@ -3489,7 +3489,7 @@ class ScheduleOverlapPolicy(IntEnum):
     """
 
     SKIP = int(
-        temporalio.api.enums.v1.ScheduleOverlapPolicy.SCHEDULE_OVERLAP_POLICY_SKIP
+        temporalio.api.enums.v1.ScheduleOverlapPolicy.SCHEDULE_OVERLAP_POLICY_SKIP  # pyright: ignore
```

Comment:
I am curious, what is the issue on this one? There is `ValueType = typing.NewType("ValueType", builtins.int)` on the proto side.

Reply:
Good call drawing attention to that. It didn't make sense as a pyright false positive to an extent that it needed looking into. The `pyright: ignore`s aren't the right solution; looks like the dataclass decorator is accidental.